src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,77 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 20px;
+  width: 340px;
+}
+
+.formInput {
+  display: flex;
+  flex-direction: row;
```

Comment:
This is redundant, since the default flex direction is `row`. I could see an argument to making it explicit, but then by that logic, we should explicitly set the flex direction for all of the `display: flex` elements in this file.

Reply:
ok